Core/UI/Utils/ObservedObject.hpp

```diff
@@ -70,6 +70,14 @@ class ObservedObject {
      */
     T get() const { return value; }
 
+    /**
+     * @brief Get a reference to the current value of the observed object.
+     * Make sure to call publish on the observed object once your changes are complete to notify listeners.
+     * This is useful when the observed object is a class.
+     * @return a mute
+     */
+    T& getMutable() { return &value; }
```

Comment:
This is how we will be able to interact with the class and then notify listeners